schemas/context/opportunity-crm.schema.json

```diff
@@ -0,0 +1,158 @@
+  "title": "CRM Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunity"
+  ],
+  "description": "CRM specific opportunity fields.",
+  "definitions": {
+    "opportunity-crm": {
+      "properties": {
+        "xdm:opportunityName": {
+          "title": "Opportunity Name",
+          "description": "Subject or descriptive name, such as the expected order or company name, for the opportunity.",
+          "type": "string"
+        },
+        "xdm:sourceType": {
+          "title": "Opportunity Source Type",
+          "description": "Opportunity source type",
+          "type": "string"
+        },
+        "xdm:description": {
+          "title": "Opportunity Description",
+          "description": "Additional information to describe the opportunity, such as possible products to sell or past purchases from the customer.",
+          "type": "string"
+        },
+        "xdm:ownerID": {
+          "title": "Opportunity Owner",
+          "description": "Assigned owner for this opportunity.",
+          "type": "string"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Source of the opportunity.",
+          "type": "string"
+        },
+        "xdm:stage": {
+          "title": "Opportunity Stage",
+          "description": "Sales stage of this opportunity to aid the sales team in their efforts to win this opportunity.",
+          "type": "string"
+        },
+        "xdm:amount": {
+          "title": "Opportunity Amount",
+          "description": "Opportunity Amount.",
+          "type": "number",
+          "minimum": 0
+        },
+        "xdm:currencyCode": {
+          "title": "Currency Code",
+          "description": "Standard ISO code(ISO 4217) for currency.",
+          "type": "string",
+          "pattern": "^([A-Z]{3})$"
+        },
+        "xdm:quantity": {
+          "title": "Opportunity Quantity",
+          "description": "Opportunity Quantity.",
+          "type": "integer",
+          "minimum": 0
+        },
+        "xdm:fiscalYear": {
+          "title": "Fiscal Year",
+          "description": "The fiscal year this opportunity is targeted.",
+          "type": "integer",
+          "pattern": "^([0-9]{4})$"
+        },
+        "xdm:fiscalQuarter": {
+          "title": "Fiscal Quarter",
+          "description": "The fiscal quarter this opportunity is targeted..",
+          "type": "string"
+        },
+        "xdm:opportunityLineItem": {
+          "title": "Has Opportunity Line Item Flag",
+          "description": "Flag to indicate whether this opportunity have more line items.",
+          "type": "boolean"
+        },
+        "xdm:probabilityPercentage": {
+          "title": "Probability Percentage",
+          "description": "Likelihood that opportunity will close, stated as a percentage.",
+          "type": "integer",
```

Comment:
Should this be number ? Percentage could be in decimals as well.

Reply:
This is not actual percentage .I have not seen probability percentage to be in decimal so far I changed this from number to integer after checking salesforce